src/components/Submit-Form/Submit-form.component.jsx

```diff
@@ -100,6 +103,22 @@ function SubmitForm({ userName, handleClose }) {
           onChange={handleUrlInputChange}
           fullWidth
         />
+
+        <Select
+          native
+          label='Language'
+          value={langName}
+          className={classes.input}
+          onChange={handleLangNameInputChange}
+          fullWidth
+        >
+        	{/*placeholder*/}
+          <option value="lang1">PHP</option>
```

Comment:
I think you should map over the languages in the config file that has the list of languages. 
{listOfLangs.map(lang => <option key={lang} value={lang}>{lang}</option>}